tests/test_unit/test_e2e.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
```

Comment:
I'm somewhat confused about this file, It's under `test_unit` but its name is test_e2e